addons/api/addon/services/resource-filter-store.js

```diff
@@ -61,7 +61,9 @@ export class ResourceFilter {
    * @return {string}
    */
   equals(key, value) {
-    return value !== null ? `"/item/${key}" == "${value}"` : null;
+    return value !== null && value !== undefined
```

Comment:
This had to be updated because the change to the resource filter can return `undefined` if `deserializedValue` is null due to the optional chaining... 🤯 

Side note: how do we feel about doing `value != null` to check for both null and undefined, I assumed we don't like that so was explicit.